website/docs/r/kv_secret_backend_v2.html.md

```diff
@@ -65,7 +65,8 @@ No additional attributes are exported by this resource.
 
 ## Import
 
-The KV-V2 secret backend can be imported using the `path`, e.g.
+The KV-V2 secret backend can be imported using its unique ID,
+the `${mount}/config`, e.g.
 
 ```
 $ terraform import vault_kv_secret_backend_v2.config kvv2/config
```

Comment:
nit: may want to rename the resource `example`, the repeating `config` makes this example a little confusing.